### ISSUE TITLE
Player accessibility improvements.

### ISSUE THREAD

**Issue by FrenjaminBanklin** [ucfopen/word-guess-materia-widget#2]
This list may grow as more issues are discovered.

- ~~The widget's welcome screen can only be dismissed by clicking on it.~~
  - This is easy enough to fix by also adding a `keypress` event listener to the welcome screen.

- ~~The submit and reset buttons do not visually indicate when they have focus.~~
  - This is a one-line CSS change to apply the same rules on focus that are applied on hover.

- ~~The submit and reset buttons are not reachable in the tab order.~~
  - Changing these buttons to submit/reset inputs and their parent element to a form solves the reachability problem and probably makes more sense to screen readers semantically. Existing event listeners need to be adjusted a bit to account for the buttons becoming inputs and to prevent default form behaviors.

- ~~The submit confirmation buttons when not all blanks have been filled in aren't reachable in the tab order.~~
  - This can be fixed by giving these buttons tab indices and toggling inert on the relevant parts of the game depending on user interactions. These buttons also need keyup event handlers since they currently only react to clicks. As a bonus, the 'What am I missing?' button could probably auto-focus the first empty blank as a convenience feature.
